README.md

```diff
@@ -94,6 +95,12 @@ first  😮 last     😮 email
 Hector 😮 Gonzalez 😮 h.g@nothing.com
 ```
 
+Add additional padding if desired with the `-p` flag.  Default is 1 space.  If less than 0, then no additional padding will be applied.
```

Comment:
What is the behavior if the value is 0?

Reply:
A value of 0 will exclude any padding surrounding the delimiter.  I will mention that on this line.